src/main/java/homework4/Bank.java

```diff
@@ -0,0 +1,40 @@
+package homework4;
+
+import java.util.HashMap;
+
+public class Bank<Integer, V extends Asset> extends HashMap<Integer, V> {
+    public final String addr;
+    V asset;
```

Comment:
private

Reply:
Do all fields private and add getters and setters - it is part of cleancode

Reply:
my bad

Reply:
asset must be private too